.circleci/config.yml

```diff
@@ -1,95 +1,36 @@
-      # Push new changes first, before adding new tests to your config.
-      - grafana-sync-orb/greet
+  orb-tools: circleci/orb-tools@10.0.1
 
 workflows:
-  # Prior to producing a development orb (which requires credentials) basic validation, linting, and even unit testing can be performed.
-  # This workflow will run on every commit
-  test-pack:
-    unless: << pipeline.parameters.run-integration-tests >>
+  verify:
     jobs:
-      - orb-tools/lint # Lint Yaml files
-      - orb-tools/pack # Pack orb source
-      - shellcheck/check:
-          dir: ./src/scripts
-          exclude: SC2148
-      # optional: Run BATS tests against your scripts
-      - bats/run:
-          path: ./src/tests
-      # Publish development version(s) of the orb.
-      - orb-tools/publish-dev:
-          orb-name: entur/grafana-sync-orb
-          context: orb-publishing # A restricted context containing your private publishing credentials. Will only execute if approved by an authorized user.
-          requires:
-            - orb-tools/lint
-            - orb-tools/pack
-            - bats/run
-            - shellcheck/check
-      # Trigger an integration workflow to test the
-      # dev:${CIRCLE_SHA1:0:7} version of your orb
-      - orb-tools/trigger-integration-tests-workflow:
-          name: trigger-integration-dev
-          context: orb-publishing
-          requires:
-            - orb-tools/publish-dev
-
-  # This `integration-test_deploy` workflow will only run
-  # when the run-integration-tests pipeline parameter is set to true.
-  # It is meant to be triggered by the "trigger-integration-tests-workflow"
-  # job, and run tests on <your orb>@dev:${CIRCLE_SHA1:0:7}.
-  integration-test_deploy:
-    when: << pipeline.parameters.run-integration-tests >>
-    jobs:
-      # Run any integration tests defined within the `jobs` key.
-      - integration-test-1
-      # Publish a semver version of the orb. relies on
-      # the commit subject containing the text "[semver:patch|minor|major|skip]"
-      # as that will determine whether a patch, minor or major
-      # version will be published or if publishing should
-      # be skipped.
-      # e.g. [semver:patch] will cause a patch version to be published.
-      - orb-tools/dev-promote-prod-from-commit-subject:
-          orb-name: entur/grafana-sync-orb
-          context: orb-publishing
-          add-pr-comment: false
-          fail-if-semver-not-indicated: true
-          publish-version-tag: false
+      - orb-tools/lint:
+          name: lint
+      - orb-tools/pack:
+          name: pack
+          source-dir: src
+      - orb-tools/publish:
+          name: publish
+          context: global
+          filters:
+            branches:
+              ignore: master
+          checkout: false
+          attach-workspace: true
+          orb-ref: entur/grafana-sync-orb@dev:$CIRCLE_BRANCH
```

Comment:
Kan vi ikke anta at "orb" er implisitt og dermed overflødig i navnet?

Reply:
This one's on me. I registered the orb with that name, but you are correct. All orbs in the registry are orbs so we could omit it.
That would require me to register it as a new orb, though,